containers/buildAlpine6.docker/tools/gcc

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/sccache gcc "$@"
```

Comment:
don't shellscripts need a final newline?

Reply:
Should not harm. Worked fine with older containers.